core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -55,20 +58,22 @@ public ListPacker(long targetWeight, int lookback) {
     private final long targetWeight;
     private final int lookback;
     private final Function<T, Long> weightFunc;
+    private final boolean expensiveTaskFirst;
 
     public PackingIterable(Iterable<T> iterable, long targetWeight, int lookback,
-                           Function<T, Long> weightFunc) {
+                           Function<T, Long> weightFunc, boolean expensiveTaskFirst) {
```

Comment:
I think this should have a better name. This name assumes that the packing is done to produce tasks, but this is more generic. A better name would be `largestBinFirst`.

Reply:
+1 Done. I carried forward the name from previous PR review.